outlines/forms.md

```diff
@@ -0,0 +1,23 @@
+# Forms and Methods
+
+1. Methods
+  1. The concept of a method and how it corresponds to a form
+  2. The `simple:method` package and the parts of a method
+  3. The basic API of a "form" method -- throwing `ValidationError`s
+  4. Using SS / C2 to build a simple form method
+2. Calling methods from the client
```

Comment:
I think this should mention that if your code is calling several methods one right after the other, it probably makes sense to make one method that does all of those things, to avoid unnecessary server roundtrips.